src/components/PeoplePicker/PeoplePicker.scss

```diff
@@ -30,19 +30,26 @@ $personaItemHeight: 42px;
   @include ms-u-normalize;
   background-color: $ms-color-white;
   margin-bottom: 10px;
+  position: relative;
 }
 
 // Box that contains the search field and selected people.
 .ms-PeoplePicker-searchBox {
   @include ms-u-clearfix;
   border: 1px solid $ms-color-neutralTertiaryAlt;
   box-sizing: border-box;
-  min-height: 40px;
+  min-height: 42px;
   width: 100%;
 
   &:hover {
     border-color: $ms-color-neutralSecondaryAlt;
   }
+
+   > .ms-FocusZone{
+    flex-flow: row wrap;
```

Comment:
Confirm use of flex. @dzearing what is your say on using flexbox? I have been personally avoiding it if it could be done some other way/